### DISCUSSION TITLE
Why brave/chromium when I open a link from a third party app opens in a new window and in a new user?

### DISCUSSION THREAD

**Discussion by zachvlat** [ScoopInstaller/Extras#9989] [Q&A]
Pretty much what the title says, I recently moved into scoop, so far so good, but I find this totally baffling. Can someone point me to the right direction. Weirdly Firefox works fine out of the box.

**Comment by rashil2000** [comment]
See https://github.com/ScoopInstaller/Extras/issues/9548 and https://github.com/ScoopInstaller/Extras/issues/9910

**Reply by zachvlat** [reply]
Yes I posted this, but I didn't understand how to make it work.

**Reply by rashil2000** [reply]
There's no way to make it work currently - it's a known bug
